Automattic/vip-go-mu-plugins
# Update Jetpack 9.5 to the final release

## Description

Update the Jetpack 9.5 subtree to the final Jetpack 9.5 release.

## Changelog Description

### Plugin Updated: Jetpack 9.5

We have upgraded the Jetpack 9.5 subtree to contain the final Jetpack 9.5 release. This will be set as the default for all unpinned sites on Thursday 4th.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Add the define `VIP_JETPACK_PINNED_VERSION` set to `9.5`
1. Go to the Jetpack dashboard in `wp-admin`
1. Ensure that the available version is 9.
1. Ensure everything else keeps working
